src/graph/entities/qg_edge.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Redis Labs Source Available License Agreement
+ */
+
+#pragma once
+
+#include "edge.h"
+#include "qg_node.h"
+#include <string.h>
+
+struct QGEdge {
+    const char *alias;
+    const char **reltypes;
+    int *reltypeIDs; // TODO can be uint save for GRAPH_NO_RELATION
```

Comment:
"SAVE for GRAPH_NO_RELATION" ?

Reply:
Relation types are unsigned ints, so that would be the preferable representation, but we need to be able to use the GRAPH_NO_RELATION and GRAPH_UNKNOWN_RELATION macros here as well. Those should probably just be changed to UINT_MAX and UINT_MAX-1 instead of -1 and -2.